### PR TITLE
Adding my profile

### PR DESCRIPTION
**Pull request by TusharAgarwal18**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1863"></a>

### CODE REVIEW

File: public/data/TusharAgarwal18.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  ],
+  "milestones": [
+    {
+      "title": "",
+      "date": "",
+      "icon": "",
+      "color": "",
+      "description": "",
+      "url": ""
+    }
```
Comment (Panquesito7) [inline public/data/TusharAgarwal18.json:26]: I recommend removing these empty fields if you're not going to use them.

Comment (TusharAgarwal18) [reply public/data/TusharAgarwal18.json]: Thanks

**Review by Panquesito7 — APPROVED** [review]
Thanks for contributing! 🎉

File: public/data/TusharAgarwal18.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  "name": "Tushar Agarwal",
+  "type": "Personal",
+  "bio": "Programmer",
+  "avatar": "https://github.com/TusharAgarwal18.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/TusharAgarwal18",
+      "icon": "github"
+    },
```
Comment (kumarsonsoff3) [inline public/data/TusharAgarwal18.json:11]: Please remove this trailing comma from line no. `11`.
```suggestion
    }
```

File: public/data/TusharAgarwal18.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  ],
```
Comment (kumarsonsoff3) [inline public/data/TusharAgarwal18.json:12]: Also from there, 
```suggestion
  ]
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Please remove the unnecessary extra comma's from line no. `11`, and `12` to avoid builds fail! 
Just make the changes as suggested below, let us know if you need any help with this! :)

**Review by EmmaDawsonDev — APPROVED** [review]
I fixed the errors for you so we can now get this merged.

**Pull request merged**